common/src/main/java/io/sphere/sdk/utils/CompletableFutureUtils.java

```diff
@@ -0,0 +1,118 @@
+package io.sphere.sdk.utils;
+
+import io.sphere.sdk.utils.functional.TriFunction;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.concurrent.*;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import org.apache.commons.lang3.tuple.Pair;
+
+import static java.util.Arrays.asList;
+
+public final class CompletableFutureUtils {
+    private CompletableFutureUtils() {
+    }
+
+    public static <T> CompletableFuture<T> successful(final T object) {
+        return CompletableFuture.completedFuture(object);
+    }
+
+    //TODO only for test scope
+    public static <T> Throwable blockForFailure(final CompletionStage<T> future) {
+        try {
+            future.toCompletableFuture().join();
+            throw new NoSuchElementException(future + " did not complete exceptionally.");
+        } catch (final CompletionException e1) {
+            return e1.getCause();
+        }
+    }
+
+    public static <T> CompletableFuture<T> failed(final Throwable e) {
```

Comment:
Yes, since the class is CompletableFutureUtils. If you want a CompletionStage, just assign it to a completion stage. :)